### PR TITLE
Review brand api default collections order

### PR DESCRIPTION
**Pull request by ironBatista**
#### Type Of Change 
- [ ] Small bug fix
- [X] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [X] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [X] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [X] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX) 
- [ ] Upport from existing Pull request #XXXX

#### Description
Reviewed Brand section from Klear and adapted new portals to ensure that it serves at least the same data.

### CODE REVIEW

File: web/rest/brand/config/api/raw/provider.yml [hunk 1]
```diff
@@ -1204,6 +1204,21 @@ Ivoz\Provider\Domain\Model\User\User:
       ROLE_BRAND_ADMIN:
         raw: 'FALSE'
 
+Ivoz\Provider\Domain\Model\MediaRelaySet\MediaRelaySet:
```
Comment (mmadariaga) [inline web/rest/brand/config/api/raw/provider.yml:1207]: MediaRelaySet is a platform entity

**Review by mmadariaga — CHANGES_REQUESTED** [review]

File: web/portal/brand/src/entities/BillableCall/BillableCall.tsx [hunk 2]
```diff
@@ -105,7 +105,7 @@ const BillableCall: EntityInterface = {
   columns: [
     'startTime',
     'company',
-    'duration',
+    'direction',
```
Comment (mmadariaga) [inline web/portal/brand/src/entities/BillableCall/BillableCall.tsx:108]: update API tests

**Review by mmadariaga — APPROVED** [review]

**Comment by ironArt3mis** [comment]
Rebase completed.

**Review by mmadariaga — CHANGES_REQUESTED** [review]
Squash commits

**Pull request merged**